### ISSUE TITLE
B0 speed and accuracy is lower than GoogleNet (Incpetion V-1)?

### ISSUE THREAD

**Issue by ivder** [lukemelas/EfficientNet-PyTorch#54]
I have tried different combination of pre-trained EfficientNet B0~B5 and compared it to pre-trained GoogleNet model. Here's the result of panda classification using the provided sample code :
```
python googlenet.py

Time = 12ms
-----
giant panda, panda, panda bear, coon bear, Ailuropoda melanoleuca           (94.11%)
Arctic fox, white fox, Alopex lagopus                                       (0.20%)
soccer ball                                                                 (0.19%)
hog, pig, grunter, squealer, Sus scrofa                                     (0.16%)
French bulldog                                                              (0.16%)

python efficientnet.py
Loaded pretrained weights for efficientnet-b0
torch.Size([1, 3, 224, 254])

Time = 50ms
-----
giant panda, panda, panda bear, coon bear, Ailuropoda melanoleuca           (88.04%)
ice bear, polar bear, Ursus Maritimus, Thalarctos maritimus                 (0.95%)
lesser panda, red panda, panda, bear cat, cat bear, Ailurus fulgens         (0.16%)
badger                                                                      (0.14%)
soccer ball
```
Tested on Gtx 1080ti, the googlenet's inference time is faster and the accuracy also higher than B0. I want to confirm whether it is true or I did something wrong when doing the classification?

**Comment by korabelnikov** [comment]
> I have tried different combination of pre-trained EfficientNet B0~B5 and compared it to pre-trained GoogleNet model. I expected the efficientNet b0 will outperform googlenet due to lesser parameters and higher accuracy. Here's the result of panda classification using the provided sample code :
> 
> ```
> python googlenet.py
> 
> Time = 12ms
> -----
> giant panda, panda, panda bear, coon bear, Ailuropoda melanoleuca           (94.11%)
> Arctic fox, white fox, Alopex lagopus                                       (0.20%)
> soccer ball                                                                 (0.19%)
> hog, pig, grunter, squealer, Sus scrofa                                     (0.16%)
> French bulldog                                                              (0.16%)
> 
> python efficientnet.py
> Loaded pretrained weights for efficientnet-b0
> torch.Size([1, 3, 224, 254])
> 
> Time = 50ms
> -----
> giant panda, panda, panda bear, coon bear, Ailuropoda melanoleuca           (88.04%)
> ice bear, polar bear, Ursus Maritimus, Thalarctos maritimus                 (0.95%)
> lesser panda, red panda, panda, bear cat, cat bear, Ailurus fulgens         (0.16%)
> badger                                                                      (0.14%)
> soccer ball
> ```
> 
> Tested on Gtx 1080ti, the googlenet's inference time is faster and the accuracy also higher than B0. I want to confirm whether it is true or Did I do something wrong when doing the classification?

Have you found the reason of low speed?
